package.json

```diff
@@ -22,18 +22,18 @@
     "build": "browserify js/index.js | uglifyjs -m > js/build.js"
   },
   "dependencies": {
-    "fruitdown": "1.0.1",
-    "hash-brown-router": "^1.3.3",
-    "levelup": "~1.2.1",
+    "fruitdown": "1.0.2",
+    "hash-brown-router": "^3.0.0",
```

Comment:
Upgrading to 3.0.0 [might be ok](https://github.com/TehShrike/hash-brown-router/blob/master/changelog.md).